sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_responses.hpp

```diff
@@ -68,40 +68,40 @@ namespace Azure { namespace Storage { namespace Blobs {
       std::string LeaseId;
     };
 
-    class StartCopyBlobResult : public Azure::Core::Operation<GetBlobPropertiesResult> {
-    public:
-      std::string RequestId;
-      Azure::Core::ETag ETag;
-      Azure::Core::DateTime LastModified;
-      std::string CopyId;
-      Models::CopyStatus CopyStatus;
-      Azure::Core::Nullable<std::string> VersionId;
+  } // namespace Models
 
-    public:
-      GetBlobPropertiesResult Value() const override { return m_pollResult; }
+  class StartCopyBlobOperation : public Azure::Core::Operation<Models::CopyStatus> {
```

Comment:
What was your rationale for changing the derived type to be `CopyStatus` instead of `GetBlobPropertiesResult`?
What happens if I want to call CopyProgress to log/monitor, while polling in a loop, how many bytes have been copied so far?

Reply:
reverted